modules/webapp/webapp.bicep

```diff
@@ -71,6 +75,14 @@ resource webApp 'Microsoft.Web/sites@2022-03-01' = {
           name: 'DBPassword'
           value: dbPassword
         }
+        {
+          name: 'redcapCommunityUsername'
+          value: redcapCommunityUsername
+        }
+        {
+          name: 'redcapCommunityPassword'
+          value: redcapCommunityPassword
```

Comment:
This probably should be a reference to KeyVault instead of saving it directly on the webapp settings. What are your thoughts @sjyang18 , @SvenAelterman , @kalalvishal ?

Reply:
Agreed. @sjyang18 Can you update the code?